src/components/ExpandedViewMode/ExpandedViewModeFooter/ExpandedViewModeFooter.css

```diff
@@ -12,21 +12,34 @@
 }
 
 .expanded_view_footer div {
-  
 }
 
-.footer_status_unknowns, .footer_leaf_completes {
+.footer_children_info {
+  display: flex;
+  flex-direction: row;
+}
+
+.footer_leaf_completes,
+.footer_status_unknowns {
   align-items: center;
   display: flex;
   margin-right: 16px;
 }
 
-.footer_children_info {
-  display: flex;
-  flex-direction: row;
+.footer_leaf_completes > .icon,
+.footer_status_unknowns > .icon {
+  margin-right: 2px;
+}
+
+.footer_leaf_completes .icon .inner-icon {
+  background-color: #83e300;
+}
+
+.footer_status_unknowns .icon .inner-icon {
+  background-color: #ff5d36;
```

Comment:
I don't like these ones... we should handle setting the icon color for Status related colors another way, perhaps using `bg_Uncertain` className or related